csharp/extractor/Semmle.Extraction/Entity.cs

```diff
@@ -153,7 +156,7 @@ public static void DefineLabel(this IEntity entity, TextWriter trapFile, IExtrac
             catch(Exception ex)  // lgtm[cs/catch-of-all-exceptions]
             {
                 trapFile.WriteLine("\"");
-                extractor.Message(new Message("Unhandled exception generating id", entity.ToString(), null, ex.StackTrace.ToString()));
+                extractor.Message(new Message("Unhandled exception generating id", entity.ToString() ?? "", null, ex.StackTrace));
```

Comment:
This appears to be a common pattern; would it make sense to have a `ToStringDefaultEmpty(this object o)` extension method?

Reply:
I think `?? ""` is extremely clear and succinct, and I suppose there would be cases where you want something other than `""`. Good suggestion though.